### PR TITLE
docs: Fix typos

### LINKED ISSUE #1062: Fix doc typos
Fix some spelling mistakes in the docs.

### PR DESCRIPTION
**Pull request by jodh-intel**
Fix some spelling mistakes in the docs.

Fixes #1062.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @klynnrif - ptal.

File: docs/architecture/architecture.md [hunk 1]
```diff
@@ -373,7 +373,7 @@ signals, `cc-runtime` will kill the `cc-shim`.  All other signals are ignored.
 #### [`delete`](https://github.com/clearcontainers/runtime/blob/master/delete.go)
 
 `delete` is about deleting all resources held by a stopped/killed container.
-Running containers can not be deleted unless the OCI runtime is explictly being
+Running containers can not be deleted unless the OCI runtime is explicitly being
```
Comment (klynnrif) [inline docs/architecture/architecture.md]: Running containers cannot be deleted...

Comment (jodh-intel) [reply docs/architecture/architecture.md]: Thanks @klynnrif - branch updated with fixes for three occurrences of that issue ;)

**Comment by devimc** [comment]
lgtm

**Pull request merged**